### PR TITLE
CMS/Pages Readability Improvements

### PR DESCRIPTION
**Pull request by DanHarrin**
Making a series of readability PRs for the whole of the documentation. These changes are minor, but improve the flow of the document.

### CODE REVIEW

File: cms-pages.md [hunk 1]
```diff
@@ -198,22 +198,22 @@ They can also be accessed in the markup using the [`this.page` variable](../mark
...
+If needed, you can inject assets (CSS and JavaScript files) into pages with the controller's `addCss` and `addJs` methods. It could be done in the `onStart` function defined in the [PHP section](themes#php-section) of a page or [layout](layout) template. Example:
 
     function onStart()
     {
         $this->addCss('assets/css/hello.css');
         $this->addJs('assets/js/app.js');
     }
 
-If the path specified in the `addCss` and `addJs` method argument begins with a slash (/) then it will be relative to the website root. If the asset path does not begin with a slash then it is relative to the theme.
+If the path specified in the `addCss` and `addJs` method argument begins with a slash (/), it will be relative to the website root. If the asset path does not begin with a slash, it is relative to the theme.
```
Comment (bennothommo) [inline cms-pages.md]: @DanHarrin IMO, I think this sentence already reads fine as is.

File: cms-pages.md [hunk 2]
```diff
@@ -198,22 +198,22 @@ They can also be accessed in the markup using the [`this.page` variable](../mark
...
 
-Injected assets can be combined by passing them in as an array:
+Injected assets can be combined by passing them as an array:
```
Comment (bennothommo) [inline cms-pages.md]: @DanHarrin I don't think `in` is required here.

**Comment by bennothommo** [comment]
@DanHarrin Looks great - I've just left a couple of comments on some of the edits which I think may not be necessary or could be edited further.

File: cms-pages.md (hunk 1 above)
Comment (DanHarrin) [reply cms-pages.md]: @bennothommo I have to stand by this edit, imagine reading the sentence out loud, you would naturally add the commas to improve flow.

Comment (bennothommo) [reply cms-pages.md]: @DanHarrin yeah, that's fair. :)

File: cms-pages.md [hunk 3]
```diff
@@ -129,12 +129,12 @@ The next example is more complicated. It shows how to load a blog post collectio
         {% endfor %}
     </ul>
 
-The default variables and Twig extensions provided by October are described in the [Markup Guide](../markup). The overall sequence the handlers are executed is described in the [Dynamic layouts](layouts#dynamic-layouts) article.
+The default variables and Twig extensions provided by October are described in the [Markup Guide](../markup). During the overall sequence, the handlers are executed as described in the [Dynamic layouts](layouts#dynamic-layouts) article.
```
[the remark this answers is not in the document]
Comment (DanHarrin) [reply cms-pages.md]: @bennothommo I've taken some of your suggestion on this one and just made it more concise.
```suggestion
The default variables and Twig extensions provided by October are described in the [Markup Guide](../markup). The sequence that the handlers are executed in is described by the [Dynamic layouts](layouts#dynamic-layouts) article.
```

**Comment by DanHarrin** [comment]
All done @bennothommo :)

**Comment by bennothommo** [comment]
Great job @DanHarrin! :)

**Pull request merged**